hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -4075,6 +4075,10 @@ public void completeMiniBatchOperations(
 
         if (!initialized) {
           this.writeRequestsCount.add(batchOp.size());
+          if (rsServices != null && rsServices.getMetrics() != null) {
+            rsServices.getMetrics().updateServerWriteQueryPerSecond(this.htableDescriptor.
```

Comment:
What about increServerWriteQuery?

Reply:
Incre is usually used for simple numerical increments, but Meter is a complex data structure. What about updateWriteQueryMeter?

Reply:
OK, i see.